core/dbt/contracts/graph/nodes.py

```diff
@@ -1783,6 +1768,17 @@ def same_config(self, old: "SavedQuery") -> bool:
     def same_group(self, old: "SavedQuery") -> bool:
         return self.group == old.group
 
+    def same_exports(self, old: "SavedQuery") -> bool:
+        if len(self.exports) != len(old.exports):
```

Comment:
Is this check here sufficient? seems like we might be updating exports but didn't realizing it?